pkg/storage/ec/client.go

```diff
@@ -261,7 +262,7 @@ func (ec *ecClient) Repair(ctx context.Context, limits []*pb.AddressedOrderLimit
 	}()
 
 	if successfulCount == 0 {
```

Comment:
Can you use here `atomic.LoadInt32(&successfulCount)`?

Reply:
Yes. I have no idea what the difference is. Can you explain that to me?

Reply:
Its making access to `successfulCount` atomic which is important when we are working with goroutines to avoid concurrent access.